pkg/notifier/telegramnotifier/interaction.go

```diff
@@ -60,14 +62,14 @@ func (it *Interaction) Session() *Session {
 }
 
 func (it *Interaction) HandleInfo(m *telebot.Message) {
-	if it.session.Owner == nil {
+	if it.session.Owner == nil || it.session.Chat == nil {
 		return
 	}
 
 	if m.Sender.ID != it.session.Owner.ID {
```

Comment:
you need to check Owner == nil here i guess?

Reply:
Yes. If (Owner == nil OR Chat == nil), then stop handling request

Reply:
got it, thanks~